README.md

```diff
@@ -51,11 +51,14 @@ returns the parsed keyfile json as a python dictionary.
 
 Takes the following parameters:
 
-- `private_key`: A bytestring of length 32
+- `private_key`: A bytestring of length 32. See note below
```

Comment:
Nit: Change ``see note`` to link to the note itself. Might need an anchor on that section for it to work.

Reply:
done!